docs/director/03-statuses.md

```diff
@@ -0,0 +1,17 @@
+# Application and Runtime status
+
+Applications and Runtimes in the Director use the `Status` field to inform the user about their condition. The possible conditions are:
+- `INITIAL` - used for newly created Applications and Runtimes that haven't performed any call to the Director
+- `PROVISIONING` - used for Runtimes that are being provisioned
+- `CONNECTED` - used for connected Applications and Runtimes
+- `FAILED` - used for Applications and Runtimes whose connection with the Director failed
```

Comment:
```suggestion
- `FAILED` - used for Applications and Runtimes whose connection with the Director failed
```

I was wondering: `connection` or `attempt to connect`?

Reply:
Changed 👍